static/app/components/acl/role.tsx

```diff
@@ -1,77 +0,0 @@
-import {useMemo} from 'react';
-
-import ConfigStore from 'sentry/stores/configStore';
-import type {Organization} from 'sentry/types/organization';
-import type {User} from 'sentry/types/user';
-import {isActiveSuperuser} from 'sentry/utils/isActiveSuperuser';
-import {isRenderFunc} from 'sentry/utils/isRenderFunc';
-import withOrganization from 'sentry/utils/withOrganization';
-
-type RoleRenderProps = {
-  hasRole: boolean;
-};
-
-type ChildrenRenderFn = (props: RoleRenderProps) => React.ReactElement | null;
-
-function checkUserRole(user: User, organization: Organization, role: RoleProps['role']) {
-  if (!user) {
-    return false;
-  }
```

Comment:
I don't know why user was here? It just checks that user exists? Was that important?

Reply:
Yeah I don't see a situation where you could render this and not have a user available